### PR TITLE
Add mirage-vnetif

### PR DESCRIPTION
**Pull request by MagnusS**

### CODE REVIEW

**Comment by MagnusS** [comment]
Hm, I'm not quite sure why this build failed. Seems to be related to conduit and tls?

**Comment by samoht** [comment]
This should be fixed by https://github.com/ocaml/opam-repository/pull/3778. I've re-launched the tests.

**Comment by MagnusS** [comment]
Failed again on conduit/tls.

File: packages/mirage-vnetif/mirage-vnetif.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+build: [
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "mirage-vnetif"]
+]
+depends: [ 
+  "lwt"
+  "mirage-types"
```
Comment (avsm) [inline packages/mirage-vnetif/mirage-vnetif.0.1/opam]: you need lower bounds on these to make sure you pick up the latest `mirage-types`

**Pull request merged**